docs/docs/1.0.1/policies/health-check.md

```diff
@@ -4,10 +4,9 @@ This policy enables Kuma to keep track of the health of every data plane proxy,
 
 By creating an `HealthCheck` resource we can instruct a data plane proxy to keep track of the health status for any other data plane proxy. When health-checks are properly configured, a data plane proxy will never send a request to another data plane proxy that is considered unhealthy. When an unhealthy data plane returns to a healthy state, Kuma will resume sending requests to it again.
 
-This policy provides the following types of checks:
+This policy provides only **active** checks. If you want to configure **passive** checks, please take a look at [Circuit Breaker](./circuit-breaker.md). **Active** means that the data plane 
```

Comment:
```suggestion
This policy provides **active** checks. If you want to configure **passive** checks, please utilize the [Circuit Breaker](./circuit-breaker.md) policy. Data plane proxies with **active** checks will explicitly send requests to other data plane proxies to determine if target data plane proxies are healthy or not. This mode will generate extra traffic to other data plane proxies and services as described in the policy configuration.
```
Trying to reduce the use of parentheses mid-sentence, thoughts?

Reply:
It totally makes sense 👍